ansible_collections/arista/avd/roles/cv_deploy/README.md

```diff
@@ -140,6 +140,15 @@ For test and lab usage the certificate verification can be disabled.
 cv_verify_certs: false
 ```
 
+For an on-premise CloudVision cluster it is possible to authenticate with username/password instead of a service account token.
+The username and password must be set via variables and `ansible_password` and `cv_token` **must not** be set.
```

Comment:
this sentence is confusing - is it:

`The username and password must be set via (variables and `ansible_password`) and `cv_token` **must not** be set.`

or

`The username and password must be set via variables (and `ansible_password` and `cv_token`) **must not** be set.`

Reply:
```suggestion
The username and password below must be set via variables on the task, play or in the fabric-level group vars. `ansible_password` and `cv_token` **must not** be set.
```